src/astro/sql/operators/agnostic_load_file.py

```diff
@@ -53,68 +46,50 @@ def execute(self, context: Any) -> Table:
         """
         Load an existing dataset from a supported file into a SQL table.
         """
-        if self.file_conn_id:
-            BaseHook.get_connection(self.file_conn_id)
+        if self.input_file.conn_id:
+            BaseHook.get_connection(self.input_file.conn_id)
 
+        database = create_database(self.output_table.conn_id)
         self.normalize_config = populate_normalize_config(
             ndjson_normalize_sep=self.ndjson_normalize_sep,
-            database=create_database_from_conn_id(self.output_table.conn_id),
+            database=database,
         )
 
-        hook = get_hook(
-            conn_id=self.output_table.conn_id,
-            database=self.output_table.metadata.database,
-            schema=self.output_table.metadata.schema,
-            warehouse=self.output_table.metadata.warehouse,
-        )
-
-        self._configure_output_table(context)
-        return self.load_data(hook=hook, path=self.path, file_conn_id=self.file_conn_id)
+        return self.load_data(database=database, input_file=self.input_file)
 
-    def load_data(
-        self, path: str, hook: BaseHook, file_conn_id: Optional[str] = None
-    ) -> Table:
+    def load_data(self, input_file: File, database: BaseDatabase) -> Table:
         """Loads csv/parquet table from local/S3/GCS with Pandas.
         Infers SQL database type based on connection then loads table to db.
         """
-        self.log.info(f"Loading {self.path} into {self.output_table}...")
+        self.log.info(f"Loading {self.input_file.path} into {self.output_table}...")
```

Comment:
```suggestion
        self.log.info("Loading %s into %s ...", self.input_file.path, self.output_table)
```

Reply:
We should probably use https://github.com/globality-corp/flake8-logging-format in pre-commit Hooks so that we always use Lazy loading of args in logs.

I did it for Providers repo a while back: https://github.com/astronomer/astronomer-providers/pull/212